src/sentry/api/endpoints/project_plugin_details.py

```diff
@@ -70,13 +71,30 @@ def post(self, request, project, plugin_id):
             plugin = self._get_plugin(plugin_id)
             plugin.reset_options(project=project)
             context = serialize(plugin, request.user, PluginWithConfigSerializer(project))
+
+            self.create_audit_entry(
+                request=request,
+                organization=project.organization,
+                target_object=project.id,
+                event=AuditLogEntryEvent.INTEGRATION_EDIT,
+                data={'integration': plugin_id, 'project': project.slug}
```

Comment:
Is there a better way to do this?

Reply:
This is fine. What did you have in mind? Or what do you think is bad here?

Reply:
Oh, then this is fine. If anything, I think there might be value in expanding on what is stored during `INTEGRATION_EDIT`. Like, information about what was edited. Otherwise, this audit log simply says "X edited an integration". But no context over what was edited, etc.

Not a blocker here, but just my two cents.

Reply:
Definitely, I'll most likely try and work on that next. Wanted to get this out as a first step since there weren't any integration audit logs at all